websites/N/Neko-sama.fr/iframe.ts

```diff
@@ -0,0 +1,10 @@
+const iframe = new iFrame();
+
+iframe.on("UpdateData", async () => {
+	const video = document.querySelector("video");
+	iframe.send({
+		time: video?.currentTime,
+		duration: video?.duration,
+		paused: video?.paused,
+	});
```

Comment:
Formatting broken here, but you might as well add this if instead
```suggestion
    if (!isNaN(video?.duration)) {
		iframe.send({
			time: video.currentTime,
			duration: video.duration,
			paused: video.paused,
		});
	}
```